jobs/config.json

```diff
@@ -2491,6 +2491,25 @@
       "UNKNOWN"
     ]
   },
+  "ci-kubernetes-e2e-gce-master-new-downgrade-cluster": {
+    "args": [
+      "--check-leaked-resources",
+      "--check-version-skew=false",
+      "--env-file=jobs/platform/gce.env",
+      "--env-file=jobs/env/ci-kubernetes-e2e-gce-new-master-upgrade-cluster.env",
```

Comment:
(intentionally reuse the same config as the upgrade env)

Reply:
Does this set the skew variables appropriately to start at master and downgrade to 1.7?

Reply:
added `--skew` flag